### ISSUE TITLE
Problem building the web image - lxml  fails to compile

### ISSUE THREAD

**Issue by swedishmike** [MISP/x_old_misp_docker#46]
When I try to build the images with `docker-compose build` it errors out with the following message.

```
Building lxml version 4.4.1.
Building without Cython.
Using build configuration of libxslt 1.1.28
ERROR: /usr/bin/xslt-config: 1: /usr/bin/xslt-config: xml2-config: not found

** make sure the development packages of libxml2 and libxslt are installed **

Compile failed: command 'x86_64-linux-gnu-gcc' failed with exit status 1
error: Setup script exited with error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
ERROR: Service 'web' failed to build: The command '/bin/sh -c python setup.py install' returned a non-zero code: 1
```
It _seems_ that the correct development libraries are being installed via the Dockerfile. I'm currently troubleshooting it, to my best availability, but thought I'd open an issue here in case it's an known issue?

**Comment by torsolaso** [comment]
HI,

i fixed it in https://github.com/MISP/misp-docker/issues/47

Regards

**Comment by swedishmike** [comment]
Fixed by adding the lines mentioned in #47
